### DISCUSSION TITLE
Exposing the current version from my npm package's CLI

### DISCUSSION THREAD

**Discussion by timrogers** [semantic-release/semantic-release#3400] [Q&A]
👋🏻 Hey there!

I'm using `semantic-release` to ship an `npm` package which exposes a CLI.

In that CLI, I'd like to have a `--version` command which returns the current version of the package.

What would be the recommended way of doing that?

I've tried using `process.env.NPM_PACKAGE_VERSION` and that does not seem to be set in the way I'd hope. 

I've also tried having a build process which loads the value from `package.json`, but it also seems that the value has not been updated at the time `npm run build` happens.

Thanks!

**Comment by travi** [comment]
i dont think your question is specific to semantic-release, by my recommendation would be to simply import the `package.json` of your package and reference the `version` property. even if you dont have that set during development because of our recommendation to avoid making commits to update it, it is still updated at publish time, so would be available and accurate at runtime after installing your published package.

**Reply by timrogers** [reply]
@travi Thanks for jumping it! Interesting...I have tried the approach of importing the `version` from `package.json`, but I'm finding that it's inaccurate (I still get `0.0.0-development`). Is there a good way I can debug that interactively without actually publishing a release?

**Reply by travi** [reply]
your `0.0.0-development` version would only be present when interacting with your unpublished package. when publishing, the version is updated in the package.json. when interacting with a published version of the package, you would get the real version
